### ISSUE TITLE
top_score_filter.py KeyError: 'Chromosome'

### ISSUE THREAD

**Issue by golubnikova** [griffithlab/pVACtools#586]
* pvactools version: 1.5.10
* Python version: Python 3.7.5

**Describe the bug**
I want to use a filter `pvacbind top_score_filter` for a file generated by `pvacbind run` (see the full commands below). When I run `pvacbind top_score_filter` , I get an error.

**Log Output**
```
Traceback (most recent call last):                                                                  
  File "/usr/local/bin/pvacbind", line 8, in <module>                                               
    sys.exit(main())                                                                                
  File "/usr/local/lib/python3.7/dist-packages/tools/pvacbind/main.py", line 58, in main            
    args[0].func.main(args[1])                                                                      
  File "/usr/local/lib/python3.7/dist-packages/tools/pvacbind/top_score_filter.py", line 11, in main
    TopScoreFilter(args.input_file, args.output_file, args.top_score_metric).execute()              
  File "/usr/local/lib/python3.7/dist-packages/lib/top_score_filter.py", line 38, in execute        
    chromosome = line['Chromosome']                                                                 
KeyError: 'Chromosome' 
```  

Input file: `YP_009725310_1.all_epitopes.tsv`
```
Mutation	HLA Allele	Sub-peptide Position	Epitope Seq	Median Score	Best Score	Best Score Method	MHCflurry Score	MHCnuggetsI Score	NetMHC Score	PickPocket Score	SMM Score	SMMPMBEC Score	cterm_7mer_gravy_score	max_7mer_gravy_score	difficult_n_terminal_residue	c_terminal_cysteine	c_terminal_proline	cysteine_count	n_terminal_asparagine	asparagine_proline_bond_count
hCoV-19/Russia/StPetersburg-3524/2020_EPI_ISL_415710.1.14	HLA-A*01:01	1	SLENVAFN	35971.383	5518.38	MHCnuggetsI	35534.415492951804	5518.38	36408.35	9550.53265102	90978.7572151	293501.280266	0.29999999999999993	0.6857142857142857	False	False	False	0	False	0
hCoV-19/Russia/StPetersburg-RII3992/2020_EPI_ISL_420080.1.14	HLA-A*01:01	1	SLENVAFN	35971.383	5518.38	MHCnuggetsI	35534.415492951804	5518.38	36408.35	9550.53265102	90978.7572151	293501.280266	0.29999999999999993	0.6857142857142857	False	False	False	0	False	0
hCoV-19/Russia/StPetersburg-RII3997/2020_EPI_ISL_420081.1.14	HLA-A*01:01	1	SLENVAFN	35971.383	5518.38	MHCnuggetsI	35534.415492951804	5518.38	36408.35	9550.53265102	90978.7572151	293501.280266	0.29999999999999993	0.6857142857142857	False	False	False	0	False	0
hCoV-19/Russia/Moscow_PMVL-1/2020_EPI_ISL_421275.1.14	HLA-A*01:01	1	SLENVAFN	35971.383	5518.38	MHCnuggetsI	35534.415492951804	5518.38	36408.35	9550.53265102	90978.7572151	293501.280266	0.29999999999999993	0.6857142857142857	False	False	False	0	False	0
```

Output file (there is only header): `YP_009725310_1.filtered_lowest.tsv`
```
Mutation	HLA Allele	Sub-peptide Position	Epitope Seq	Median Score	Best Score	Best Score Method	MHCflurry Score	MHCnuggetsI Score	NetMHC Score	PickPocket Score	SMM Score	SMMPMBEC Score	cterm_7mer_gravy_score	max_7mer_gravy_score	difficult_n_terminal_residue	c_terminal_cysteine	c_terminal_proline	cysteine_count	n_terminal_asparagine	asparagine_proline_bond_count
```

In the script [top_score_filter.py](https://github.com/griffithlab/pVACtools/blob/master/lib/top_score_filter.py#L37) I found a check  string: 
```
if self.file_type != 'pVACbind':
    chromosome = line['Chromosome']
      …...…..
```
Could it be the reason for my error? If yes, could you correct it, please?
                                                                         
**Commands**
`pvacbind top_score_filter`
`pvacbind top_score_filter -m lowest /output/pvacbind_results/YP_009725310.1/MHC_Class_I/YP_009725310_1.all_epitopes.tsv /output/pvacbind_results/YP_009725310.1/MHC_Class_I/YP_009725310_1.filtered_lowest.tsv`

 `pvacbind run`
```
pvacbind run -e 8,9,10,11 --iedb-install-directory "$IEDB_PATH" \
--allele-specific-binding-thresholds -m lowest --n-threads 4 \
/input/YP_009725310.1.fasta YP_009725310_1 \
'HLA-A*02:01,HLA-A*03:01,HLA-A*01:01,HLA-A*24:02,HLA-A*11:01,HLA-B*07:02,HLA-B*18:01,HLA-B*35:01,HLA-B*08:01,HLA-B*13:02,HLA-C*07:02,HLA-C*04:01,HLA-C*07:01,HLA-C*06:02,HLA-C*12:03' \
MHCflurry MHCnuggetsI MHCnuggetsII NNalign NetMHC PickPocket SMM SMMPMBEC SMMalign "/output/test/"
```

With best wishes,
Liliya

**Comment by susannasiebert** [comment]
Thank you for your error report. It's being addressed by #584 which will be released with the next hotfix release.

**Comment by susannasiebert** [comment]
I apologize for the delay but we finally have a new hotfix release (version 1.5.11) that fixes this issue.
